static/src/javascripts/bootstraps/enhanced/youtube.js

```diff
@@ -0,0 +1,92 @@
+define([
+    'fastdom',
+    'common/modules/video/youtube-player',
+    'common/modules/video/youtube-tracking',
+    'common/utils/$'
+], function (
+    fastdom,
+    youtubePlayer,
+    tracking,
+    $
+) {
+
+    var STATES = {
+        'ENDED': _onPlayerEnded,
+        'PLAYING': _onPlayerPlaying,
+        'PAUSED': _onPlayerPaused
+    };
+
+    var progressTracker = {};
+    var players = {};
+
+    function init() {
+
+        fastdom.read(function () {
+                        $('.atom--media--youtube').each(function (el) {
+                            var atomId = el.getAttribute('data-media-atom-id');
+                            var videoId = el.firstElementChild.id;
+                            tracking.init(atomId);
```

Comment:
is this right? `init` seems to want a video id - https://github.com/guardian/frontend/pull/14965/files#diff-76e86efc7805cf2a7c9429d5276bf49cR86